Makefile

```diff
@@ -1,20 +1,23 @@
+.PHONY: unittest
+
 deps-docs:
 	go install kubepack.dev/chart-doc-gen@latest
 
 docs: deps-docs
 	find . -name "doc.yaml" | \
 		xargs -L1 dirname | \
 		xargs -I% sh -c \
-			"cd %; chart-doc-gen -v values.yaml -d doc.yaml -t README.tpl > README.md"
+			"chart-doc-gen -c %/Chart.yaml -v %/values.yaml -d %/doc.yaml -t %/README.tpl > %/README.md"
 
 lint:
 	find . -name "Chart.lock" -type f -delete
 	docker run --rm -e CT_VALIDATE_MAINTAINERS=false -u $(shell id -u) -v $(PWD):/charts quay.io/helmpack/chart-testing:latest sh -c "cd /charts; ct lint --all"
 
-deps-helm:
-	helm plugin install https://github.com/quintush/helm-unittest  || true
-
-test-unit-all: deps-helm test-registry-scanner
+deps-unittest:
+	@helm plugin install https://github.com/helm-unittest/helm-unittest --version=0.3.0 || true
 
-test-registry-scanner:
-	helm unittest --helm3 ./charts/registry-scanner
+unittest: deps-unittest
```

Comment:
non blocking feedback.

i prefer the makefile to be the main orchestrator for both pre-commit and github worklows to invoke it.
also, any reason on why the makefile has a generic  helm unittest, but the github workflow one invocation per chart?

Reply:
@wideawakening I agree on having a single source of through and not having different definitions (e.g.: Makefile Vs GH Actions Vs ...) but not sure what will be the unified SSoT we are going to use here (I think should be a discussion that we need to start soon :D ). I'll keep you in the loop if you're interested in the discussion 🙌